src/Starcounter.Database.Extensions/PostCommitTransactorContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using Starcounter.Database.ChangeTracking;
+
+namespace Starcounter.Database.Extensions
+{
+    public class PostCommitTransactorContext
```

Comment:
A good candidate to be a protected nested class - nobody cares about it.

Reply:
- Nested class - works.
- `protected` - does not, it has to be public due to the usage: `public class PostCommitTransactor : OnCommitTransactor<PostCommitTransactor.PostCommitTransactorContext>`.

I don't mind moving it inside `PostCommitTransactor`, if you think that's better.

Reply:
>protected - does not, it has to be public due to the usage

Then having it nested is not that appealing. Leave it as you want

Reply:
Okay, leaving as is.